gammapy/background/fov.py

```diff
@@ -3,3 +3,53 @@
 Field-of-view (FOV) background estimation
 """
 from __future__ import print_function, division
+from astropy.wcs import WCS
+from astropy.wcs.utils import pixel_to_skycoord
+from scipy.interpolate import interp1d
+
+__all__ = ['fill_acceptance_image',
+           ]
+
+
+def fill_acceptance_image(image, center, offset, acceptance):
+    """Generate a 2D image of a radial acceptance curve given as
```

Comment:
Try to have a one-line description ... it then shows up nicer in the overview docs.
In this case.
```
Fill acceptance image from radial acceptance curve.

... detailed description if needed ...
```